test/Utilities/copy.jl

```diff
@@ -653,6 +653,7 @@ struct OnlyCopyConstraints{F,S} <: MOI.ModelLike
         return new{F,S}(MOIU.VectorOfConstraints{F,S}())
     end
 end
+function MOI.Utilities._add_variable(::OnlyCopyConstraints) end
```

Comment:
If this is a public API that people need to implement, it shouldn't begin with `_`.

Reply:
This needs to be implemented by the type of the `constraints` field but it's currently not public API.
It's best we don't let user create new types at the moment until we converge. This is similar to `_delete_variables`, `_deleted_constraints` and `_throw_if_cannot_delete`.